cmd/rm-main.go

```diff
@@ -121,17 +133,32 @@ func checkRmSyntax(ctx *cli.Context) {
 	isForce := ctx.Bool("force")
 	isRecursive := ctx.Bool("recursive")
 	isStdin := ctx.Bool("stdin")
-
+	isDangerous := ctx.Bool("dangerous")
+	isNamespaceRemoval := false
+	for _, url := range ctx.Args() {
+		if !isAliasURLDir(url) && (strings.Count(url, string(filepath.Separator)) == 0) {
```

Comment:
1. Please add a comment about meaning of this logic.
2. `()` around `strings.` is not required.

Reply:
done